core/src/main/scala/org/broadinstitute/dsde/rawls/metrics/InstrumentationDirectives.scala

```diff
@@ -9,7 +11,7 @@ trait InstrumentationDirectives extends RawlsInstrumented {
     val timeStamp = System.currentTimeMillis
     mapHttpResponse { response =>
       httpRequestCounter(ExpandedMetricBuilder.empty)(request, response).inc()
-      httpRequestTimer(ExpandedMetricBuilder.empty)(request, response).time(System.currentTimeMillis - timeStamp)
+      httpRequestTimer(ExpandedMetricBuilder.empty)(request, response).update(System.currentTimeMillis - timeStamp, TimeUnit.MILLISECONDS)
```

Comment:
@ansingh7115 @jmthibault79 my bad for not catching this in review. `Timer.time()` times the block of code passed to it. Here's the signature:
```
def time[A](f: => A): A
```
whereas `Timer.update()` adds a recorded duration. Signature:
```
 def update(duration: Long, unit: TimeUnit): Unit
```
So the previous code wasn't really correct in that it was timing the actual code `System.currentTimeMillis - timeStamp`.